### PR TITLE
new methods in class object,integer,numeric,string

### PR DESCRIPTION
**Pull request by olhoscastanhosrj**
Some methods that I think useful to have on primitive types.

Text translate using google translate.

### CODE REVIEW

**Comment by olhoscastanhosrj** [comment]
Methods that use along with rails and can be useful for you but someone.

Texto traduzido pelo google translate.

File: activesupport/lib/active_support/core_ext/string/envelop.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		self.envelop('[' , ']')
+	end
+
+	#this string between braces
+	def braces 
+		self.envelop('{' , '}')
+	end
+
+	#this string between slashs
+	def slashs
```
Comment (sh6khan) [inline activesupport/lib/active_support/core_ext/string/envelop.rb]: the plural of slash is slashes

Comment (olhoscastanhosrj) [inline activesupport/lib/active_support/core_ext/string/envelop.rb]: Thanks for the answer.
Now I hit?

Text translated by google translate.

**Comment by olhoscastanhosrj** [comment]
Updated. Compare.rb the file has been fixed.
The methods that are also in envelop.rb file were fixed.

Text translated by google translate.

**Comment by rafaelfranca** [comment]
Thank you so much for the pull request but we don't accept new methods on Active Support core extensions.

**Comment by olhoscastanhosrj** [comment]
Ok, thanks for the feedback.

Text translated by google translate.

**Pull request closed without merging**